### ISSUE TITLE
App Support to install NPM peerDependencies

### ISSUE THREAD

**Issue by adeelibr** [netlify/netlify-cms#2059]
**Is your feature request related to a problem? Please describe.**

I have a repo that manages the src/ code of the library along with the documentation portal code as well. In my case I can't have some packages in my dependencies section of package.json. 

So I need to put them in my peerDependencies section, but when I execute the `build` command, the peerDependencies don't get installed & my build crashes.

**Describe the solution you'd like**
Add a build option that gives the option to install peerDependencies.

**Describe alternatives you've considered**

* The alternative of this would be to manually install react & react-dom before the build ran everytime. Which doesn't seem an elegant solution

* Or have peerDependencies as devDependencies, which isn't semantically correct as well.

**Comment by erquhart** [comment]
Having a hard time understanding your use case. You're maintaining a fork of Netlify CMS?

**Comment by adeelibr** [comment]
Sorry bad choice of words, I had a React library that I was bundling with rollup, I had to set the `NODE_ENV` to `development` in my netlify build so that the dev dependencies would have installed.

The problem has been solved.

This might have not been the appropriate place to ask this question. :) Stalkoverflow was 😅 

Reference to my question here https://stackoverflow.com/questions/54502468/configuring-rollup-for-a-react-library/54507972#54507972
